azure-sql/migration-guides/modernization.md

```diff
@@ -282,11 +282,11 @@ See the [Performance](/azure/azure-sql/managed-instance/frequently-asked-questio
 
 #### I want to modernize SQL Server workloads to Azure SQL. What is the next step?
 
-A great place to start is joining the [Azure Migration and Modernization Program](https://azure.microsoft.com/migration/migration-modernization-program/#overview). When you start a migration project, a good practice is to form a dedicated Migration team to formulate and execute the migration plan. If your company has an assigned Microsoft or Microsoft Partner account team, they can provide guidance regarding Migration team required skill set and overall process.
+A great place to start is joining the [Azure Migration and Modernization Program](https://azure.microsoft.com/migration/migration-modernization-program/#overview). When you start a migration project, a good practice is to form a dedicated Migration team to formulate and execute the migration plan. If your company has an assigned Microsoft or Microsoft Partner account team, they can provide guidance regarding the Migration team required skill set and overall process.
 
 #### Where can I find migration guides to Azure SQL?
 
-The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/) that also contains guides for migrating to another database targets.
+The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/) which also contains guides for migrating to another database targets.
```

Comment:
In our style guide, "which" always takes a preceding comma, whereas "that" doesn't.

```suggestion
The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/), which also contains guides for migrating to another database targets.
```